reporting_database_generator/elastic_scripts/09_13_elastic_publication_publication.sql

```diff
@@ -0,0 +1,52 @@
+{% set reporting_db = "pregrant_publications" %}
+
+use `{{elastic_db}}`;
+
+CREATE TABLE IF NOT EXISTS `{{elastic_db}}`.`publication`
+(
+    `id`                                                    varchar(128) NOT NULL,
+    `pgpubs_document_number`                                varchar(20) COLLATE utf8mb4_unicode_ci NOT NULL,
+    `type`                                                  varchar(100) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+    `application_number`                                    varchar(64) COLLATE utf8mb4_unicode_ci NOT NULL,
+    `country`                                               varchar(20) COLLATE utf8mb4_unicode_ci  DEFAULT NULL,
+    `date`                                                  date                                    DEFAULT NULL,
+    `year`                                                  smallint(5) unsigned DEFAULT NULL,
+    `abstract`                                              text COLLATE utf8mb4_unicode_ci         DEFAULT NULL,
+    `title`                                                 text COLLATE utf8mb4_unicode_ci         DEFAULT NULL,
+    `kind`                                                  varchar(10) COLLATE utf8mb4_unicode_ci  DEFAULT NULL,
+    `series_code`                                           varchar(64) COLLATE utf8mb4_unicode_ci NOT NULL,
+    `rule_47_flag`                                          boolean NOT NULL,
+    PRIMARY KEY (`id`),
+    KEY                                                     `ix_publication_number` (`pgpubs_document_number`),
+    KEY                                                     `ix_publication_application_number` (`application_number`),
+    KEY                                                     `ix_publication_title` (`title`(128)),
+    KEY                                                     `ix_publication_type` (`type`),
+    KEY                                                     `ix_publication_year` (`year`),
+    KEY                                                     `ix_publication_date` (`date`),
+    KEY                                                     `ix_publication_country` (`country`),
+    FULLTEXT KEY `fti_publication_title` (`title`),
+    FULLTEXT KEY `fti_publication_abstract` (`abstract`)
+) ENGINE = InnoDB
+  DEFAULT CHARSET = utf8mb4
+  COLLATE = utf8mb4_unicode_ci;
+
+TRUNCATE TABLE `{{elastic_db}}`.`publication`;
+
+insert into `{{elastic_db}}`.publication ( id, pgpubs_document_number, type, application_number, country, date, year, abstract, title, kind, series_code, rule_47_flag)
+select p.id
+     , p.document_number
+     , a.type
+     , a.application_number
+     , p.country
+     , p.date
+     , YEAR(p.date) as year
+     , a.invention_abstract
+     , a.invention_title
+     , p.kind
+     , a.series_code
+     , a.rule_47_flag
+from `{{reporting_db}}`.publication p
+    left join `{{reporting_db}}`.application a
+on a.document_number = p.document_number;
```

Comment:
@csoule-air  For each of the queries in this PR, can you generate an [explain plan](https://dev.mysql.com/doc/refman/8.0/en/explain.html)?